deltachat-ios/Controller/SettingsController.swift

```diff
@@ -20,15 +38,163 @@ internal final class SettingsViewController: QuickTableViewController {
         return hudHandler
     }()
 
+    // MARK: - cells
+
+    private let profileHeader = ContactDetailHeader()
+
+    private lazy var profileCell: ProfileCell = {
+        let displayName = DcConfig.displayname ?? String.localized("pref_your_name")
+        let email = (DcConfig.addr ?? "")
+        let selfContact = DcContact(id: Int(DC_CONTACT_ID_SELF))
+        let cell = ProfileCell(contact: selfContact)
+        cell.tag = CellTags.profile.rawValue
+        return cell
+    }()
+
+    private var contactRequestCell: UITableViewCell = {
```

Comment:
what is the rationale behind the fact that some cells are marked as lazy and others not?

Reply:
I removed two unneeded 'lazy' statements.

By declaring objects lazy you 'delay' the instantiation of the corresponding object. It is needed in case the corresponding object/code block uses methods / attributes of the surrounding class (which need to be instantiated first).

As far as I know a lazy statement does not harm.